usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -163,6 +164,14 @@ def f(provider):
     return f
 
 
+@pytest.fixture()
+def get_baseproduct():
+    """Return the nmae of the file the 'baseproduct' link points to"""
+    baseproduct = '/etc/products.d/baseproduct'
+    if os.path.islink(baseproduct):
```

Comment:
This would run on the client machine (the machine running img-proof). Need to use the `host` commands to run this on the server being tested. See [is_sles_sap](https://github.com/SUSE-Enceladus/img-proof/blob/master/usr/share/lib/img_proof/tests/SLES/conftest.py#L351) for example.